src/inc/msquic.h

```diff
@@ -105,6 +110,13 @@ typedef enum QUIC_CONNECTION_SHUTDOWN_FLAGS {
 
 DEFINE_ENUM_FLAG_OPERATORS(QUIC_CONNECTION_SHUTDOWN_FLAGS);
 
+typedef enum QUIC_SEND_RESUMPTION_FLAGS {
+    QUIC_SEND_RESUMPTION_FLAG_NONE          = 0x0000,
+    QUIC_SEND_RESUMPTION_FLAG_FINAL         = 0x0001    // Free TLS state after sending this ticket.
```

Comment:
I think we should we have a flag for 0-RTT key too.

Reply:
I imagined 0-RTT (read and write) being controlled by the resumption level instead. Are you suggesting that the 0-RTT resumption level (the session param) should instead control whether the server accepts 0-RTT (read) and this flag controls whether the server will send 0-RTT keys (write)?

Reply:
You're right. Ignore this comment.